### ISSUE TITLE
Verrucous epidermal nevus 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7366]
**Preferred term label:**
Verrucous epidermal nevus 

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of epidermal nevus (which represent Blaschkoid hamartomas of the skin that result from mosaic post-zygotic mutations) that appears as skin-colored-to-brown, sharply demarcated, papillomatous papules that coalesce into plaques. The majority of epidermal nevi are either present at birth or occur within the first year of life
PMID:24364998

**Parent term (use hpo.jax.org/app)**
Epidermal nevus HP:0010816

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added

**Comment by carolina-f** [comment]
32155026[PMID]_29267480[PMID]
Woolly hair nevus ORPHA 79414
